docs/tutorial.ipynb

```diff
@@ -368,7 +368,125 @@
+    "We are going to create the model twice, once on the raw data and again on the synthetic data, and see how the predictive power\n",
+    "of the model varies between the two approaches.\n",
+    "\n",
+    "First, we need to split the raw data into a training dataset and a test dataset. Furthermor, we'll separate the attribute columns from the target column:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "raw_data_test = data.frame.iloc[:80, :]\n",
+    "raw_data_test_y = raw_data_test[\"target\"]\n",
+    "raw_data_test_X = raw_data_test.drop(columns=[\"target\"])\n",
+    "\n",
+    "raw_data_train = data.frame.iloc[80:, :]\n",
+    "raw_data_train_y = raw_data_train[\"target\"]\n",
+    "raw_data_train_X = raw_data_train.drop(columns=[\"target\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, we'll fit a model on the raw training dataset and use it to predict the `target` column:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.40482699584417636"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import sklearn.linear_model, sklearn.metrics\n",
+    "\n",
+    "model = sklearn.linear_model.LinearRegression().fit(raw_data_train_X, raw_data_train_y)\n",
+    "sklearn.metrics.r2_score(raw_data_test_y, model.predict(raw_data_test_X))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's create a synthetic dataset from the raw training data and then repeat the previous process:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.21163884597541138"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "syn_data_train = Synthesizer(raw_data_train).sample()\n",
+    "syn_data_train_y = syn_data_train[\"target\"]\n",
+    "syn_data_train_X = syn_data_train.drop(columns=[\"target\"])\n",
+    "\n",
+    "model = sklearn.linear_model.LinearRegression().fit(syn_data_train_X, syn_data_train_y)\n",
+    "sklearn.metrics.r2_score(raw_data_test_y, model.predict(raw_data_test_X))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see, the predictive power of the model is significantly lower.\n",
+    "Fortunately, we can pass a focus column to the `Synthesizer` which will create a synthetic dataset better suited for ML tasks:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.29617271972853276"
```

Comment:
Hmm, the score isn't much higher. Ideally we'd want a dataset where the raw score is good and the syn score is close to it...